guides/Masternode_Upgrade_Cold_Hot_Linux.md

```diff
@@ -0,0 +1,129 @@
+
+If you are using Windows, [PuTTY](https://putty.org) is a very good SSH client that you can use to connect to a remote Linux server.
+
+### 2. Stop the service with:
+```
+shekel-cli stop
+```
+
+### 3. Upgrade the Shekel CLI wallet. Always download the latest [release available](https://github.com/shekeltechnologies/JewNew/releases), unpack it
+#### For Ubuntu 14.04 (see below for 16.04)
+```
+apt-get update -y
+apt-get upgrade -y
+apt-get install libzmq3 libminiupnpc-dev -y
+https://github.com/shekeltechnologies/JewNew/releases/download/1.3.0.0/shekel-linux-1.3.0.zip
+unzip shekel-linux-1.3.0.zip
+rm shekel-linux-1.3.0.zip
+chmod +x shekel-cli shekeld
+mv shekel-cli shekeld /usr/local/bin/
+```
+#### For Ubuntu 16.04 only
+```
+apt-get update -y
+apt-get upgrade -y
+apt-get install libzmq3-dev libminiupnpc-dev -y
+https://github.com/shekeltechnologies/JewNew/releases/download/1.3.0.0/shekel-Ubuntu16.04-1.3.0.zip
+unzip shekel-Ubuntu16.04-1.3.0.zip
+rm shekel-Ubuntu16.04-1.3.0.zip
+chmod +x shekel-cli shekeld
+mv shekel-cli shekeld /usr/local/bin/
+```
+
+### 4. Start the service with:
+```
+shekeld
+```
+
+### 8. Wait until is synced with the blockchain network:
+Run this command every few mins until the block count stopped increasing fast.
+```
+shekel-cli getinfo
+``` 
+Give it 30 mins now for this node to "get social" with the other nodes in the network. Once it peers up with a good number of other masternodes, the following activation steps should work fine.
+
+
+## Re-Enable the Masternode
+
+### 1. Go back to the local(cold) wallet and open `Tools` > `Debug console`.
+#### ** Note, you might have to unlock the wallet first if you have a passphrase ***
+Type this command to see all the MasterNodes loaded from the `masternode.conf` file with their current status:
+```
+masternode list-conf
+```
+
+You should now see the newly added MasterNode with a status of `MISSING`.
+
+Run the following command, in order to enable it:
+```
+startmasternode alias false MN1
+```
+In this ^ case, the alias of my MasterNode was MN1, in your case, it might be different.
+
+
+### 2. Verify that the MasterNode is enabled and contributing to the network.
+
+Give it a few minutes and go to the Linux VPS console and check the status of the masternode with this command:
+```
+shekel-cli masternode status
+```
+
+If you see status `Masternode successfully started`, you've done it, congratulations. Go hug someone now :)
+It will take a few hours until the first rewards start coming in.
+
+Instead, if you get status `Masternode not found in the list of available masternodes`, you need a bit more patience. Distributed systems take a bit of time to reach consensus. Restarting the wallets and retrying the start has been reported to help by community members. This is how you restart the Linux wallet from the CLI:
+```
+shekel-cli stop
+# wait 30 seconds or so for the wallet to gracefully stop and then start it again
```

Comment:
I've seen this take 1-2 minutes. Let's make it 60 seconds.

Quick way to check if it has stopped:
```
ps aux | grep shekeld
```

Reply:
Yeah, good idea